benchmark/src/main/java/com/amazon/opendistroforelasticsearch/sql/benchmark/utils/load/mysql/MysqlDataLoader.java

```diff
@@ -31,6 +51,64 @@
    */
   @Override
   public void loadData(DataFormat data) throws Exception {
+    if (!(data instanceof MysqlDataFormat)) {
+      throw new IllegalArgumentException("Wrong data format for MySQL.");
+    }
+
+    Class.forName("com.mysql.cj.jdbc.Driver");
+    connection = DriverManager.getConnection(url);
+    statement = connection.createStatement();
+
+    createDatabase();
+    createTables();
+
+    Map<String, LinkedList<String>> list = ((MysqlDataFormat) data).getTableDataFilesList();
+
+    for (String table : list.keySet()) {
+      for (String file : list.get(table)) {
+        String query =
+            "LOAD DATA LOCAL INFILE '" + file + "' INTO TABLE " + MysqlTpchSchema.databaseName + "."
+                + table + " FIELDS TERMINATED BY ',' ENCLOSED BY '\"'"
+                + " LINES TERMINATED BY '\n'";
+        statement.executeUpdate(query);
+      }
+    }
+  }
+
+  private void createDatabase() throws Exception {
+    statement.executeUpdate("SET GLOBAL local_infile=1;");
+    statement.executeUpdate("create database " + MysqlTpchSchema.databaseName);
+  }
+
+  private void createTables() throws Exception {
+    for (String tablename : MysqlTpchSchema.schemaMap.keySet()) {
+      String sql = "create table " + MysqlTpchSchema.databaseName + "." + tablename + " (";
+      for (String field : MysqlTpchSchema.schemaMap.get(tablename).keySet()) {
+        sql += field + " " + MysqlTpchSchema.schemaMap.get(tablename).get(field) + ", ";
+      }
+
+      sql += "primary key ( ";
+      int index = 1;
+      for (String key : MysqlTpchSchema.primaryKeyMap.get(tablename)) {
```

Comment:
Perhaps instead use StringJoiner (https://docs.oracle.com/javase/8/docs/api/java/util/StringJoiner.html) to do this.

Reply:
added